.github/workflows/main.yml

```diff
@@ -7,107 +7,21 @@ on:
       - master
 
 jobs:
-  # Enforce the standard of g_malloc instead of malloc in the project
-  enforce-g_malloc:
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v2
-      with:
-        # Clone the whole branch, we have to fetch tags later
-        fetch-depth: 0
-
-    - name: enforce-g_malloc
-      run: "! git grep -P '(?<!g_)malloc' ':!.valgrind.suppressions' ':!.github/workflows/main.yml'"
```

Comment:
This is super important, that devs use GLib here and not the libc defaults. It does not relate to cmake/meson, so no reason to remove.

Reply:
In which codebase is this specific to? Dunst? I thought that because dunst never used any 'malloc' calls and was using 'g_malloc' it was already done without needing to change.

Reply:
Right. It's for those who are unaware of it. So the CI will fail and automatically tell about their misuse.